sdk/confidentialledger/Azure.Security.ConfidentialLedger/src/ConfidentialLedgerCertificateClientOptions.cs

```diff
@@ -29,7 +31,7 @@ public ConfidentialLedgerCertificateClientOptions(ServiceVersion version = Lates
         {
             Version = version switch
             {
-                ServiceVersion.V2022_05_13 => "2022-05-13",
+                ServiceVersion.V2024_01_26_Preview => "V2024_01_26_Preview",
```

Comment:
Shuoldn't this be `2024-01-26-preview` ?

Reply:
Yes you're right, fixed this.  It appears this file is used for the identity endpoint (there's another identical file for the ledger endpoints), we don't even really need this file right?

Reply:
Yeah I agree it may not be needed since it's not dependent on the API version, but we could keep it for now just to reduce the number of changes in this PR :)